src/middlewares/verifyToken.js

```diff
@@ -27,18 +27,25 @@ export default (request, response, next) => {
 
   jwt.verify(token, secret, async (error, decoded) => {
     if (error) {
-      const message = (error.name === 'TokenExpiredError') ? 'token expired, you have to be signed in to continue' : 'you have to be signed in to continue';
-      return responseMessage(response, 401, { error: message });
-    }
-    try {
-      const user = await findUser(decoded.id);
-      if (!user) {
-        return responseMessage(response, 404, { error: 'user not found' });
+      const message = (error.name === 'TokenExpiredError') ? 'token expired' : 'invalid token';
```

Comment:
can we avoid nested if statements? nested if statements can be difficult to read. Can we do something like this instead:
```
if (error) {
  const message = ....
  return responseMessage()
}

try {} catch {}
```

Reply:
This is actually not my line of code. I included mine from line 34,35,36. It's just a middleware that checks for blacklisted token in the database.

Reply:
It's not your line of code but it will be nice if you fix it. Always leave your codebase better than you meet it

Reply:
Alright. Fixed